fold-and-r.csl

```diff
@@ -129,6 +129,15 @@
             </group>
           </group>
         </else-if>
+        <else-if type="speech">
+          <group delimiter=" ">
+            <text macro="title" prefix="'" suffix="'"/>
```

Comment:
No, I don't believe we've had that chat. :-)

The journal's guidelines call for single quote marks. So would that then be:

`<text macro="title" quotes="true"/>`

in the appropriate groups, along with the locale change up top? That seems to do it for me.

I also noted the guidelines call for a leading "in" before the journal title, so I added that too.